src/plugins/ramps/utils/getRateFromRampQuoteResult.ts

```diff
@@ -0,0 +1,20 @@
+import { div, toFixed } from 'biggystring'
+
+import { formatNumber } from '../../../locales/intl'
+import { lstrings } from '../../../locales/strings'
+import type { RampQuoteResult } from '../rampPluginTypes'
+
+export const getRateFromRampQuoteResult = (
+  quote: RampQuoteResult,
+  fiatCode: string
+): string => {
+  const bestRate = div(quote.fiatAmount, quote.cryptoAmount, 16)
+  const localeRate = formatNumber(toFixed(bestRate, 0, 2))
```

Comment:
Use our existing formatting. Refer to `useCryptoText` `useFiatText` for context

Reply:
What am I looking for in those implementations? I copied this from our previous fiat plugins exchange rate display logic. Those other hooks look to do a lot, so I don't know what I'm looking for in those components. Do you know what specifically is missing from this implementation of display logic?